project/ws/app/src/lib/routes/training-plan/routes/create-content/create-content.component.ts

```diff
@@ -106,20 +113,49 @@ export class CreateContentComponent implements OnInit {
     this.handleSelectedChips(true)
   }
 
+  // showAddContentDialog() {
+  //   this.queryParams = {
+  //     name: 'trainingPlan',
+  //   }
+  //   this.router.navigate(['/app/home/create-request-form'],{ queryParams: this.queryParams })
+  //   // const dialogRef = this.dialog.open(AddContentDialogComponent, {
+  //   //   maxHeight: 'auto',
+  //   //   height: '60%',
+  //   //   width: '60%',
+  //   // })
+  //   // dialogRef.afterClosed().subscribe((response: any) => {
+  //   //   if (response) {
+  //   //     if (response.data.responseCode === 'OK') {
+  //   //       this.snackbar.open('Request shared successfully')
+  //   //     } else {
+  //   //       this.snackbar.open('Something went wrong please try again later!!')
+  //   //     }
+  //   //   }
+  //   // })
+  // }
+
   showAddContentDialog() {
-    const dialogRef = this.dialog.open(AddContentDialogComponent, {
-      maxHeight: 'auto',
-      height: '60%',
-      width: '60%',
+    /* tslint:disable */
+    this.dialogRef = this.dialog.open(ConfirmationBoxComponent, {
+      disableClose: true,
+      data: {
+        type: 'conformation',
+        icon: 'radio_on',
+        title: 'You have unsaved progress on your CBP plan. Clicking "Yes" will discard it and take you to request new content screen. Would you like to continue?',
```

Comment:
Hardcoded data should come from constants

Reply:
as we have only hardcoded value so as of now created a global variable and pass